examples/pages/dynamic_layer_control.py

```diff
@@ -49,8 +49,8 @@
     "dashArray": "5, 5",
 }
 
-polygon_folium1 = folium.GeoJson(data=gdf1, style_function=lambda x: style_parcels)
-polygon_folium2 = folium.GeoJson(data=gdf2, style_function=lambda x: style_buildings)
+polygon_folium1 = folium.GeoJson(data=gdf1, style_function=lambda _x: style_parcels)
```

Comment:
Any specific reason to prefix this with an underscore?

Reply:
Because they're not actually used in the lambda at all.

It's equivalent to:

```
def foo(x):
    do_something()
```

So, the `x` isn't used at all. *But*, folium does pass *something* to style_function, apparently, so you can't remove it entirely. This just indicates that it's an unused function argument.